### PR TITLE
update zh_cn language

### PR DESCRIPTION
**Pull request by xiaomalover**
update zh_cn language

### CODE REVIEW

File: resources/space/spacepicker.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ $.fn.spacepicker = function (options) {
         searchUrl: "",
         currentValue: "",
         templates: {
-            inputStructure: '<div class="space_picker_container"><ul class="tag_input" id="space_tags"><li id="space_tag_input"><input type="text" id="space_input_field" class="tag_input_field" value="" autocomplete="off" placeholder="Add a space"></li></ul><ul class="dropdown-menu" id="spacepicker" role="menu" aria-labelledby="dropdownMenu"></ul></div>'
+            inputStructure: '<div class="space_picker_container"><ul class="tag_input" id="space_tags"><li id="space_tag_input"><input type="text" id="space_input_field" class="tag_input_field" value="" autocomplete="off" placeholder="增加一个空间"></li></ul><ul class="dropdown-menu" id="spacepicker" role="menu" aria-labelledby="dropdownMenu"></ul></div>'
```
Comment (luke-) [inline resources/space/spacepicker.js]: Can you please fix this?

**Comment by luke-** [comment]
Great!

**Comment by luke-** [comment]
Can you please accept our CLA?

https://www.humhub.org/cla/github/sign

**Comment by xiaomalover** [comment]
@luke- I have agreed

**Comment by buddha87** [comment]
Sorry a bit late, could you resolve the conflicts and send another pull request or if you want to contribute in translating let me know, we have an extra translation community.

**Pull request closed without merging**